docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1246,6 +1246,50 @@ The output of the query has the following columns:
     - ``array(integer)``
     - The set of field IDs used for equality comparison in equality delete files
 
+``$refs`` table
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think these are supposed to match width. @mosabua might want to take a look at the docs addition, but thanks for adding it

Reply:
Updated to match the width